app/assets/stylesheets/content/_work_packages_table.sass

```diff
@@ -129,6 +129,7 @@ table.workpackages-table
       &.checkbox
         min-width: 0
         width: 20px
+        padding: 7px 5px 3px 7px
```

Comment:
I am not sure whether this is the correct approach:

1. By this change every table row has an increased height
2. I can't find the padding values anywhere else in the table row, which is at least a bad sign
3. At least according to the indicator lines of the IE you still did not match the rest of the row exactly ;-)

### Alignment of the checkbox:
![check-align](https://cloud.githubusercontent.com/assets/453584/8002967/5a23b3a0-0b73-11e5-913c-f2cfb6c381ac.png)

### Alignment of a text in the same row:
![good-align](https://cloud.githubusercontent.com/assets/453584/8002968/5a268ce2-0b73-11e5-9024-bc23e0e00344.png)